src/components/Resources/useResources.tsx

```diff
@@ -343,7 +355,14 @@ export const useResources = ({ txCostsMedianUsd }): DashboardSection[] => {
   const scalingBoxes: DashboardBox[] = [
     {
       title: t("page-resources-roadmap-title"),
-      // TODO: Add metric
+      metric: (
+        <div className="grid place-items-center py-5">
+          <div className="text-sm">Next upgrade</div>
+          <div className="text-5xl font-bold">Pectra</div>
+          {/* TODO: Convert date to a countdown */}
+          <div className="text-xl font-bold text-body-medium">07 May 2025</div>
```

Comment:
Any suggestions on which way to go here? Kinda already doing an approach in `useResources` for "Time to next block". Use that approach for this?

There is also something more robust like the [react-countdown](https://github.com/ndresx/react-countdown) component.